web/src/portico/help.ts

```diff
@@ -80,15 +88,22 @@ function scrollToHash(simplebar) {
     }
 }
 
-const cache = new Map();
-const loading = {
+type Article = {
+    html: string;
+    title: string;
+};
+
+const cache = new Map<string, Article>();
+const loading: {
+    name: string | null;
+} = {
```

Comment:
Can we probably change this to a simple variable like `current_loading_path: string | null` instead of an object given it has only one property `name` ?

Reply:
don't really know why the previous authors wrote a single var in loading object. Let me try to research on this and see if it's possible to flatten it.

Reply:
yeah, seems no obvious reason to make that const complicated. Changed it to a simple string as suggested in a separate commit, thanks !